src/studio/src/designer/frontend/packages/schema-editor/src/components/SchemaInspector.tsx

```diff
@@ -102,8 +107,9 @@ const SchemaInspector = ((props: ISchemaInspectorProps) => {
   const [title, setItemTitle] = React.useState<string>('');
   const [objectType, setObjectType] = React.useState<string>('');
   const [arrayType, setArrayType] = React.useState<string>('');
```

Comment:
Could these be more specific? Know it's outside scope, but it's a bit redundant to have both `<string>` and assignment to `''` when a more specific type is available.

Reply:
Added some typings where I saw fit + typings for actions in slice. I think we have to take it a little step by step.